package.json

```
@@ -30,11 +30,13 @@
 	},
 	"peerDependencies": {
 		"@angular/animations": "7.x",
+		"@angular/cdk": "^7.x",
```

Entweder `7.x` oder `^7.0.0`
```suggestion
		"@angular/cdk": "7.x",
```